routes/api/auth/index.js

```diff
@@ -70,20 +70,15 @@ const HandleAuthPopupCallback = (req, res, next) => (err, user) => {
     return res.render('auth-callback', {err: JSON.stringify(errors.ErrNotAuthorized), data: null});
   }
 
-  // Authorize the user to edit their username.
-  UsersService.toggleNameEdit(user.id, true)
-    .then(() => {
-      
-      // Perform the login of the user!
-      req.logIn(user, (err) => {
-        if (err) {
-          return res.render('auth-callback', {err: JSON.stringify(err), data: null});
-        }
-
-        // We logged in the user! Let's send back the user data.
-        res.render('auth-callback', {err: null, data: JSON.stringify(user)});
-      });
-    });
+  // Perform the login of the user!
+  req.logIn(user, (err) => {
+    if (err) {
+      return res.render('auth-callback', {err: JSON.stringify(err), data: null});
+    }
+
+    // We logged in the user! Let's send back the user data.
+    res.render('auth-callback', {err: null, data: JSON.stringify(user), changeusername: true});
```

Comment:
It is not evident that this is a sign in versus a sign up, so passing the `changeusername: true` doesn't exactly make sense here.